doc/contributor/howto-guide-generated-code-maintenance.md

```diff
@@ -15,6 +15,35 @@ fork:
 git checkout -b feat-websecurityscanner-generate-library
 ```
 
+Generate the library:
```

Comment:
Can/Should we remove other other set-set below?

Reply:
I want to keep a separate example showing how to generate libraries with ad-hoc directories. But the example I have was not great, maybe the new one makes more sense?